sdks/python/apache_beam/runners/portability/fn_api_runner/translations.py

```diff
@@ -1475,6 +1482,12 @@ def only_element(iterable):
   return element
 
 
+def only_transform(stage):
+  # type: Stage -> beam_runner_api_pb2.PTransform
```

Comment:
14:14:31 apache_beam/runners/portability/fn_api_runner/translations.py:1485: error: syntax error in type comment 'Stage -> beam_runner_api_pb2.PTransform'  [syntax]
14:14:31 apache_beam/runners/portability/fn_api_runner/translations.py:1485: note: Suggestion: wrap argument types in parentheses
14:14:31 Found 1 error in 1 file (checked 719 source files)
14:14:31 error: mypy exited with status 2
14:14:32 ERROR: InvocationError for command /home/jenkins/jenkins-slave/workspace/beam_PreCommit_PythonLint_Commit/src/sdks/python/test-suites/tox/py37/build/srcs/sdks/python/target/.tox-py37-mypy/py37-mypy/bin/python setup.py mypy (exited with code 1)
14:14:32 py37-mypy run-test-post: commands[0] | bash /home/jenkins/jenkins-slave/workspace/beam_PreCommit_PythonLint_Commit/src/sdks/python/test-suites/tox/py37/build/srcs/sdks/python/scripts/run_tox_cleanup.sh
14:14:32 ___________________________________ summary ____________________________________
14:14:32 ERROR:   py37-mypy: commands failed

Reply:
Fixed.